### PR TITLE
Set ramp opacity

### LINKED ISSUE #35: Ramp from ramp.nut is transparent
Somehow it only has 220 alpha. This is a bug. It should be fully opaque.

(Maybe we can show player ramps as opaque, but other players ramps as transparent?)

### PR DESCRIPTION
**Pull request by dracc**
Add a second set of ramps for opacity management.
Set the players UniqueWorld ramp at full opacity to make it distinguishable from the "public" ramp with a slight transparency.

Closes #35

### CODE REVIEW

File: scripts/module/ramp.nut [hunk 1]
```diff
@@ -24,22 +28,31 @@ function onPlayerGameKeysChange( player, oldKey, newKey ) {
       angle2 = Vector(0, 0, angle);
     }
     pos = GetForwardPoint( pos, angle, 10.0 );
-    local obj = CreateObject( 562, player.World, pos, 220 );
-    obj.RotateToEuler(angle2, 0);
-    RampObj[player.ID] = obj;
+    local publicRampObj = CreateObject( 562, player.World, pos, 177 );
```
Comment (JayFoxRox) [inline scripts/module/ramp.nut]: I commented on the other issue about 177 being possibly too high. Ideally I'd like to see a screenshot of this and a lower value.

**Comment by JayFoxRox** [comment]
Closes #35

File: scripts/module/ramp.nut (hunk 1 above)
Comment (dracc) [reply scripts/module/ramp.nut]: I'll attach that screenshot tonight!

**Comment by dracc** [comment]
[177 opacity](https://i.imgur.com/D40Dkyg.png) - You were right, way too high.
[150 opacity](https://i.imgur.com/pps66Lq.png) - Much better!
[130 opacity](https://i.imgur.com/kHnpMFx.png) - Maybe a bit low?

**Comment by JayFoxRox** [comment]
150 is the sweet spot, thanks!

**Pull request merged**